### PR TITLE
Display records using views

### PR DESCRIPTION
**Pull request by youmari**
## In this branch I implemented the following : 
- Added views for the following : 
    - users to display all the users
    - for a specific user to display user information and his/her posts
    - all the posts for a specific user
    - all comments for a specific post

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @youmari, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] **You have submitted a project, that has failed linter check. Kindly fix this.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,18 @@
...
+      <article>
+        <h2><%= post.title %></h2>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments_counter || 0 %></p>
+        <p>Likes: <%= post.likes_counter || 0 %></p>
+      </article>
+    </li>
+  <% end %>
+</ul>
+<button type="button" >See all posts<button>
```
Comment (dasileker) [inline app/views/users/show.html.erb]: - [ ] Hello @youmari nice job so far however, the button to see all the posts and comments is not rendered correctly. I can't see my posts or comments.

Capture web_29-3-2022_85145_localhost

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @youmari,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**

- Nice PR 👌.
- Linters are passing 👌.
- Good Readme file 👌.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (Emmyn5600) [reply app/views/users/show.html.erb]: - [x] This issue is not yet implemented or fixed, please fix it before you request another review.

Comment (youmari) [reply app/views/users/show.html.erb]: I implemented @Emmyn5600

Comment (Emmyn5600) [reply app/views/users/show.html.erb]: As suggested by the previous reviewer, the button or link to see all the posts and comments is not rendered correctly. I can't see my posts or comments.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @youmari 

Good job so far! I know you have worked hard on making this milestone however, There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:

- All linters are passing :heavy_check_mark:
- Good PR description :heavy_check_mark:


## Required Changes :recycle:

- [ ]  Check the comment under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by youmari** [comment]
@Emmyn5600 Please we need to talk

**Comment by youmari** [comment]
image
image

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @youmari,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**